internal/auth0/client/resource.go

```diff
@@ -481,6 +481,42 @@ func NewResource() *schema.Resource {
 				Description: "List containing a map of the public cert of the signing key and the public cert " +
 					"of the signing key in PKCS7.",
 			},
+			"addons": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				Description: "Addons enabled for this client and their associated configurations.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"aws": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							MaxItems:    1,
+							Description: "AWS Addon configuration.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"principal": {
+										Description: "AWS principal ARN, e.g. `arn:aws:iam::010616021751:saml-provider/idpname`.",
```

Comment:
```suggestion
										Description: "AWS principal ARN, for example `arn:aws:iam::010616021751:saml-provider/idpname`.",
```

Reply:
The docs team's guidelines instruct not to use latin abbreviations.

Reply:
These are exactly the ones taken from https://auth0.com/docs/api/management/v2, however I agree that `for example` reads better.